ivy_tests/test_ivy/helpers/available_frameworks.py

```diff
@@ -8,7 +9,10 @@ def _available_frameworks():
     for backend in ["numpy", "jax", "tensorflow", "torch", "paddle"]:
         if find_spec(backend) is not None:
             ret.append(backend)
-    return ret
+        elif os.path.exists(f"/opt/fw/{backend}"):
+            ret.append(backend)
+
+    return ["numpy", "jax", "tensorflow", "torch"]
```

Comment:
again, ideally this shouldn't be hardcoded.

Reply:
This was to adjust both the multiversion as well as  usual testing, we can refactor this a bit to come up with better logic.